app/helpers/alias.py

```diff
@@ -0,0 +1,10 @@
+import uuid
+
+def create_alias(input_string):
+    new_string = ''.join(
+        letter for letter in input_string.lower()
+        if letter.isalnum() or letter == '-')
```

Comment:
@inno-asiimwe didnt get what this If does?

Reply:
ensures alias is all lowercase and removes any special characters apart from -